psi4/src/psi4/dfmp2/mp2.cc

```diff
@@ -3639,6 +3648,10 @@ void RODFMP2::print_header()
     outfile->Printf( "\t %7s %7d %7d %7d %7d %7d %7d\n", "ALPHA", focc_a, occ_a, aocc_a, avir_a, vir_a, fvir_a);
     outfile->Printf( "\t %7s %7d %7d %7d %7d %7d %7d\n", "BETA", focc_b, occ_b, aocc_b, avir_b, vir_b, fvir_b);
     outfile->Printf( "\t --------------------------------------------------------\n\n");
+    
+    if (avir_a == 0 || avir_b == 0) {
+      throw PSIEXCEPTION("Number of active virtual orbitals is zero. Correlation energy = 0. a.u.");
+    }
```

Comment:
You might want to edit this slightly to imply that both alpha and beta spins need virtual orbitals. Hard to imagine a case unless someone does Li in a 2s basis set though.